### ISSUE TITLE
Redirecting the terminal output to clipit is not working

### ISSUE THREAD

**Issue by brainplot** [CristianHenzel/ClipIt#83]
Hello,

I want to pipe to output of a command to clipit in order to have it in my history and paste it wherever I need but it doesn't work. From the clipit's man page:
<pre>
<b>CLI EXAMPLES</b>
    Put text in the clipboard
       echo "copied to clipboard" | clipit
       clipit "copied to clipboard"

    Put text in the clipboard and print the clipboard contents
        echo "copied to clipboard" | clipit -c
</pre>

However, I've tried several ways to achieve that but it doesn't seem to work. So far I've tried:
* `echo "Hello!" | clipit`
* `echo "Hello!" | clipit -c   # it succesfully outputs "Hello!" though`
* `clipit "Hello!"`
* `clipit > echo "Hello!"`

All of the above commands result in my clipboard being somehow empty. If I hit <kbd>CTRL</kbd>+<kbd>V</kbd> in any text field, or <kbd>CTRL</kbd>+<kbd>SHIFT</kbd>+<kbd>V</kbd> in my terminal, it pastes nothing.
It doesn't work if I right click on a text field either because the "Paste" option in the menu is greyed out as if my clipboard is empty, as stated earlier.
If I manually select any text though, both in a text field and in my terminal, it'll work as normal.

I installed Clipit from the Arch User Repository through [this package](https://aur.archlinux.org/packages/clipit/).

How can I solve this? Hope I'm not doing anything wrong.

### System info:
__OS:__ Arch Linux x86_64
__Host:__ HP Pavilion 15 Notebook PC 0881100000305E00000620100
__Kernel:__ 4.14.12-1-ARCH
__WM: i3__
__Theme:__ Vimix-Dark-Ruby [GTK2/3]
__Icons:__ Papirus-Dark [GTK2/3]
__Terminal:__ tilix
__CPU:__ AMD A6-5200 APU (4) @ 2.000GHz
__GPU:__ AMD Radeon HD 8670A/8670M/8690M
__GPU:__ AMD Radeon HD 8400

**Comment by wallace11** [comment]
You may want to pipe to  `xclip -selection c` in the meanwhile.

**Comment by brainplot** [comment]
Thank you, that doesn't really solve the problem however.

If I'm not mistaken, I don't have `xclip` installed at all; and installing it just for this little feature, when I already have a clipboard manager, kind of bugs me.

**Comment by mgiagante** [comment]
Having the same issue on Manjaro.

**Comment by sljunkie** [comment]
I'm also having the same issue.

Ideapad 330s, Manjaro x86_64, i3, urxvt.

**Comment by andreiboyanov** [comment]
+1. I'm with Manjaro and i3, clipit is installed by default but not working from the command line. Is it possible that this bug has no attention for more than two years?

**Comment by Zeioth** [comment]
+1, I'd say this is quite important.

**Comment by emanuelturis** [comment]
Try piping the results into "echo "xsel -i -b".

**Comment by emanuelturis** [comment]
Try piping the results with xsel instead of clipit.

For example `echo "Hello!" | xsel -b`.

It worked for me with Manjaro i3 Community Edition.

**Comment by brainplot** [comment]
@emanuelturis Yeah, sure there are ways around this but since this is `clipit`'s issue tracker, it made sense to report the issue anyway. It's good to have your suggestion here for people who come here looking for a workaround though!
